src/Microsoft.Identity.Client/Utils/StringExtensions.cs

```diff
@@ -16,5 +17,72 @@ public static byte[] ToByteArray(this string stringInput)
         {
             return new UTF8Encoding().GetBytes(stringInput);
         }
+
```

Comment:
question:  are you concerned about memory fragmentation here for just doing a "while string.contains(oldValue) { string.replace(oldValue, newValue) } type of algorithm?  these strings aren't super long and don't have a lot of templatization so just using the existing string replace instead of building it up via stringbuilders might be less error prone

Reply:
I'll get something simpler.